mloq/api.py

```diff
@@ -180,7 +263,21 @@ def docs_project(
     config: DictConfig,
     overwrite: bool = False,
 ) -> None:
-    """Initialize the project folder structure and all the documentation files."""
+    """
+    Initialize the project folder structure and all the documentation files.
+
+    It calls the documentation setup method to generate (on the target
+    path) the documentation files required by the project. This folder
+    will contain configuration files (requirement lists, 'make' commands,
+    sphinx configurations) needed for building the documentation of
+    the project.
+
+    Args:
+         path: Target folder where the generated files will be written.
+         config: Configuration list summarizing the options selected for
```

Comment:
```suggestion
         config: DictConfig containing the selected project configuration
```
It's better to no mention the word `list`, given that a list is a different data structure, and `DictConfig` behaves like a dictionary.